tests/scripts/test-ref-configs.pl

```diff
@@ -65,6 +65,7 @@ sub abort {
     print "\n******************************************\n";
     print "* Testing configuration: $conf\n";
     print "******************************************\n";
+    $ENV{MBEDTLS_TEST_CONFIGURATION} = "$conf";
```

Comment:
Should this be `-$conf` like other scripts?

Reply:
No. Contrast:
* “Testing configuration: $conf” → `$conf`
* “Testing without thing: $thing” → `"-$thing"`

After double-checking, I'll fix “Testing with key exchange”.

Reply:
Ah, the `-` is for some set calculus. It means "without". I had read it as something that may get concatenated with another string, with `-` as a separator.